applications/acdc/doc/architecture.md

```diff
@@ -4,7 +4,7 @@ Agents represent the endpoints that a call in a queue will try to connect with.
 
 #### Agents
 
-Agents are comprised of two processes, a `gen_listener` (named process) and a `gen_statem`. A supervisor ensures that the `gen_listener` and `gen_statem` are kept alive together. A supervisor above that manages the list of agents.
+Agents are comprised of two processes, a `gen_listener` (named process) and a `gen_fsm`. A supervisor ensures that the `gen_listener` and `gen_fsm` are kept alive together. A supervisor above that manages the list of agents.
```

Comment:
gen_fsm is deprecated, FYI. It will be removed in future OTP versions. I would suggest reverting any gen_fsm-related changes

Reply:
This is understood and we plan to address this in a 2nd phase of work.